### PR TITLE
Added $WOSMO (SEI-INJ-TERRA-NEUTRON-ARCHWAY) -IBC

### PR DESCRIPTION
**Pull request by Dizz02**

### CODE REVIEW

File: injective/assetlist.json [hunk 1]
```diff
@@ -175,6 +175,46 @@
...
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/WOSMO.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/WOSMO.png"
+        }
```
Comment (JeremyParish69) [inline injective/assetlist.json:216]: Please use image_sync, pointing osmosis, WOSMO

File: neutron/assetlist.json [hunk 2]
```diff
@@ -747,6 +747,46 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/retro.svg"
         }
       ]
+    },
+    {
+      "description": "The first memecoin on osmosis.",
+      "denom_units": [
+        {
+          "denom": "ibc/7DA39F5140741177846FCF3CFAB14450EE7F57B7794E5A94BEF73825D3741958",
+          "exponent": 0,
```
Comment (JeremyParish69) [inline neutron/assetlist.json:756]: This comma without a property following will break the schema

File: injective/assetlist.json [hunk 3]
```diff
@@ -175,6 +175,46 @@
...
+        },
+        {
+          "denom": "WOSMO",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/DD648F5D3CDA56D0D8D8820CF703D246B9FC4007725D8B38D23A21FF1A1477E3",
+      "name": "Wosmo",
+      "display": "Wosmo",
```
[the remark this answers is not in the document]
Comment (Dizz02) [reply injective/assetlist.json]: Done (added archway too)

File: injective/assetlist.json (hunk 1 above)
Comment (Dizz02) [reply injective/assetlist.json:217]: Done (added archway too)

File: neutron/assetlist.json (hunk 2 above)
Comment (Dizz02) [reply neutron/assetlist.json]: Done (added archway too)

**Pull request merged**